WORKSPACE

```
@@ -135,7 +135,7 @@ http_archive(
 
 load("@rules_jvm_external//:defs.bzl", "maven_install")
 
-ANDROID_LINT_VERSION = "26.6.2"
+ANDROID_LINT_VERSION = "27.2.1"
```

```suggestion
ANDROID_LINT_VERSION = "30.0.0"
```

This was introduced in lint API 30 as far as I know, not 27